en/backup-by-ebs-snapshot-across-multiple-kubernetes.md

```diff
@@ -0,0 +1,404 @@
+---
+title: Back Up a TiDB Cluster across Multiple Kubernetes Using EBS Volume Snapshots
+summary: Learn how to back up TiDB cluster data across multiple Kubernetes to S3 based on EBS volume snapshots using BR Federation.
+---
+
+# Back Up a TiDB Cluster across Multiple Kubernetes Using EBS Volume Snapshots
+
+This document describes how to back up the data of a TiDB cluster deployed across multiple AWS Kubernetes clusters to AWS storage using EBS volume snapshots.
+
+The backup method described in this document is implemented based on CustomResourceDefinition (CRD) in [BR Federation](br-federation-architecture.md#br-federation-architecture-and-processes) and TiDB Operator. [BR](https://docs.pingcap.com/tidb/stable/backup-and-restore-overview) (Backup & Restore) is a command-line tool for distributed backup and recovery of the TiDB cluster data. For the underlying implementation, BR gets the backup data of the TiDB cluster, and then sends the data to the AWS storage.
+
+## Usage scenarios
+
+If you have the following requirements when backing up TiDB cluster data, you can use TiDB Operator to back up the data using volume snapshots and metadata to Amazon S3:
+
+- Minimize the impact of backup, such as keeping the impact on QPS and transaction latency less than 5%, and not utilizing cluster CPU and memory.
+- Back up and restore data in a short period of time. For example, completing a backup within 1 hour and restore it within 2 hours.
+
+If you have any other requirements, refer to [Backup and Restore Overview](backup-restore-overview.md) and select an appropriate backup method.
+
+## Prerequisites
+
+> storage blocks on volumes that were created from snapshots must be initialized (pulled down from Amazon S3 and written to the volume) before you can access the block. This preliminary action takes time and can cause a significant increase in the latency of an I/O operation the first time each block is accessed. Volume performance is achieved after all blocks have been downloaded and written to the volume.
```

Comment:
```suggestion
The storage blocks on volumes that are created from snapshots must be initialized (pulled down from Amazon S3 and written to the volume) before you can access the block. This initialization process takes time and might cause significantly increased latency for I/O operations when accessing each block for the first time. It is only after all blocks have been downloaded and written to the volume that optimal volume performance is achieved.
```

Reply:
this is referenced from AWS documentation, we shouldn't change it.

Reply:
Better to make it as an quote